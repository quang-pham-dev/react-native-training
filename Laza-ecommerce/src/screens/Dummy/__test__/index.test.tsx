import React from 'react';

// LIBS
import renderer from 'react-test-renderer';

// Screen
import DummyScreen from 'screens/Dummy/Dummy';

describe('Dummy Screen', () => {
  test('renders correctly', () => {
    const tree = renderer.create(<DummyScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
