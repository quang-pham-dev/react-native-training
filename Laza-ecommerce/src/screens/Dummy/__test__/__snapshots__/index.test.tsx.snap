// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dummy Screen renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    APP VERSION NOT YET DEVELOPED
  </Text>
</View>
`;
